docs/user_guide/data_model.rst

```diff
@@ -138,28 +138,28 @@ The JSON structure below shows the 'design' parameter exported by the :meth:`.wr
 
     "design": {
         "defvalue": null,
-        "example": [
-            "cli: -design hello_world",
-            "api: chip.set('design', 'hello_world')"
-        ],
-        "help": "Name of the top level module or library. Required for all\nchip objects.",
-        "lock": "false",
+        "lock": false,
+        "node": {
+            "global": {
+                "global": {
+                    "value": "hello_world"
+                }
+            }
+        },
```

Comment:
Might be worth mentioning that this nesting is to support per-node values?

Reply:
I noticed there is a different section that discusses that (but it's out of date)